apps/documentation/README.md

```diff
@@ -5,13 +5,7 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 First, run the development server:
 
 ```bash
-npm run dev
-# or
 yarn dev
-# or
-pnpm dev
-# or
-bun dev
```

Comment:
we can improve this doc file over time, but just going to simplify it to match the tooling we use